symfony/plugins/orangehrmLeavePlugin/Controller/LeaveRequestController.php

```diff
@@ -35,6 +43,16 @@ public function preRender(Request $request): void
             die;
```

Comment:
*

Reply:
we have to remove this `die` and throw record not found exception.
can you check `die` in other controllers